test/PrivateComposerInstaller/RemoteFilesystemTest.php

```diff
@@ -17,6 +18,10 @@ protected function setUp(): void
 
     public function testExtendsComposerRemoteFilesystem()
     {
+        if (PluginInterface::PLUGIN_API_VERSION === '2.0.0') {
```

Comment:
![kép](https://user-images.githubusercontent.com/952007/93004524-0e06d280-f548-11ea-9d2c-35014c198245.png)

@mcaskill There is a GitHub feature to request code change: the +/- toolbar button.

Reply:
Thanks @szepeviktor, for this snippet I couldn't add my comment to the `setUp()` method since its outside of the changeset. I'll update the other comments.

Reply:
You could add suggestions outside of the changeset.
See the one below.